critiquebrainz/frontend/views/review.py

```diff
@@ -198,6 +198,10 @@ def create():
         return redirect(url_for('search.selector', next=url_for('.create')))
     return render_template('review/write.html', form=form, entity_type=entity_type, entity=entity)
 
+    if review.entity_type == 'release_group':
```

Comment:
Also you should use `entity_type` there, not `review.entity_type`. Please test your changes and make sure they work before submitting.